client/src/ui/BitbakeCommands.ts

```diff
@@ -50,6 +50,7 @@ export function registerBitbakeCommands (context: vscode.ExtensionContext, bitba
     vscode.commands.registerCommand('bitbake.collapse-list', async () => { await collapseActiveList() }),
     vscode.commands.registerCommand('bitbake.start-toaster-in-browser', async () => { await startToasterInBrowser(bitBakeProjectScanner.bitbakeDriver) }),
     vscode.commands.registerCommand('bitbake.stop-toaster', async () => { await stopToaster(bitBakeProjectScanner.bitbakeDriver) }),
+    vscode.commands.registerCommand('bitbake.generate-dependency-graph', async (uri) => { await generateDependencyGraph(bitbakeWorkspace, bitBakeProjectScanner, uri) }),
```

Comment:
I have tested the integration with "yocto project dependency visualizer". The experience wasn't good. It could not find my .dot file by itself. Graphviz crashed because it requires specific settings and that extension is no longer maintained. IMO this command does not produce an easy way to get a GUI of dependencies. I suggest not keeping this commit.

Reply:
Done.

Yocto Project Dependency Visualizer expects the .dot file to be in a folder named "build" on the workspace's root.